src/allmydata/frontends/sftpd.py

```diff
@@ -1925,7 +1925,11 @@ def writeSequence(self, data):
     def loseConnection(self):
         logmsg("FakeTransport.loseConnection()", level=NOISY)
 
-    # getPeer and getHost can just raise errors, since we don't know what to return
+    def getHost(self):
+        pass
+
+    def getPeer(self):
+        pass
```

Comment:
Should we go with ... `raise NotImplementedError()` or something similar here?  I doubt any code was relying on an exception coming out of these before but .... :shrug: